hostprocess/eventflow-logger/.gitignore

```diff
@@ -0,0 +1,398 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Do we need to check in the gitignore if we are building the app in a container?

Reply:
I guess it doesn't hurt to leave it in

Reply:
if you are dev'ing locally you'll need this otherwise files will get commited